created landing page and altered button component

klevun7 opened the pull request:
<img width="1470" alt="Screenshot 2024-01-21 at 10 29 14 PM" src="https://github.com/acm-ucr/rishi-website/assets/147557875/9b46b652-6028-4b7e-bed8-fefae72b8c68">

- created landing page and altered Button.jsx to handle props of `buttonText`
- I need some help with positioning the image and the elements without the use of setting margins and padding I think it has something to do with the width and height I set for the entire component.

pondohoo commented on src/components/home/Landing.jsx:
```diff
@@ -1,5 +1,53 @@
+        </div>
+      </div>
+
+      <div className="flex relative w-screen -top-24 -right-24">
```
justify-center items-center so the image shrinks towards the middle when you shrink the screen
(adjust -top-24 -right-24 to look the same as it does now)

pondohoo commented on src/components/home/Landing.jsx:
```diff
@@ -1,5 +1,53 @@
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen ">
+      <div className="flex flex-col w-7/12 justify-center gap-3 pl-12">
```
u can honestly make it w-8/12 so the largest text doesn't wrap until smaller screen sizes

klevun7 commented:
<img width="1470" alt="Screenshot 2024-01-22 at 11 45 39 PM" src="https://github.com/acm-ucr/rishi-website/assets/147557875/740f7b21-e579-4747-86c7-4ed7c4bc0392">
- implemented changes to landing image to shrink towards the center of the screen.
- adjusted width of text to not wrap on smaller screens.
- adjusted padding of outline around image to fit closer to figma spec.
- still have problem of image moving as left margin of text div is changed.

pondohoo commented on src/components/home/Landing.jsx:
```diff
@@ -1,5 +1,53 @@
+          <Image src={logo} alt="rishi-logo" className="w-24 h-auto" />
+        </div>
+
+        <div className="flex justify-center">
```
you don't need to wrap the "Project RISHI at UCR" container in this extra flex container. delete this extra flex container and just add text-center to the Project RISHI at UCR container

pondohoo commented on src/components/home/Landing.jsx:
```diff
@@ -1,5 +1,53 @@
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen">
```
add ml-6 here to prevent it from hitting the border of the screen

klevun7 commented:
<img width="1470" alt="Screenshot 2024-01-25 at 1 19 21 PM" src="https://github.com/acm-ucr/rishi-website/assets/147557875/0e4eb16c-6a8b-4cf9-bad2-fd14e9262c17">

- removed extra flex-container around text.
- added ml-6 to prevent hitting border of screen.
